posts/2020-09-29-microprofile-openapi-beta-200011.adoc

```diff
@@ -0,0 +1,247 @@
+---
+layout: post
+title: "MicroProfile OpenAPI 2.0 plus new features and loads of improvements for Jakarta EE 9 in Open Liberty 20.0.0.11-beta"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: MicroProfile OpenAPI 2.0 plus new features and loads of improvements for Jakarta EE 9 in Open Liberty 20.0.0.11-beta - OpenLiberty.io
+seo-description: Open Liberty 20.0.0.11-beta comes with MicroProfile OpenAPI 2.0 which allows users to generate documentation directly from API's, plus two new features for Jakarta EE 9.
+blog_description: Open Liberty 20.0.0.11-beta comes with MicroProfile OpenAPI 2.0 which allows users to generate documentation directly from API's, plus two new features for Jakarta EE 9.
+---
+= New features and loads of improvements for Jakarta EE 9 plus MicroProfile OpenAPI 2.0 in Open Liberty 20.0.0.11-beta
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+In this beta release the All Beta Features package contains our new MicroProfile OpenAPI 2.0 which allows you to generate documentation directly from API's. In Jakarta EE 9 we have two features and also include improvements to features including many moving up to the latest RC's of API's. 
+
+We have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<allbeta, All Beta Features>>: a new, larger package that contains all Open Liberty beta and GA features and functions.
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools (either the All Beta Features package or the Jakarta EE 9 Beta Features package, depending on what you want to try).
```

Comment:
I don't think you need the bit in brackets ... I think that should be clear enough from the words above that describe two packages.

Reply:
Done